### ISSUE TITLE
Upstream: OWSave now hard-codes FileFormats that are permitted to save

### ISSUE THREAD

**Issue by stuart-cls** [Quasars/orange-spectroscopy#317]
See https://github.com/biolab/orange3/pull/3545 

@markotoplak perhaps you can ask if this is intentional, or just an oversight? I'm all for "read liberally, write conservatively" but...

Also I guess we weren't testing round trips on the FileFormats that do support writing.

**Comment by markotoplak** [comment]
I think is an oversight.

**Comment by markotoplak** [comment]
If I am not mistaken, Save widget was fixed to the same formats before the change, but I see no reason what it could not be extended.

**Comment by stuart-cls** [comment]
Hmm, my mistake! I think for exchanging data with other software we should at least support the .dat column-wise format.

**Comment by markotoplak** [comment]
Yes, I agree. I think that Save should show all file formats that can write.

**Comment by stuart-cls** [comment]
This is tracked upstream in https://github.com/biolab/orange3/issues/3701

Once that is implemented, we should add tests for the write function and round-trips.

**Comment by markotoplak** [comment]
This was fixed upstream and will be released with Orange 3.25.
